lib/SelectWallet.js

```diff
@@ -27,18 +37,32 @@ const styles = {
 class SelectWallet extends Component {
   constructor(props) {
     super(props);
+    this.loadingAccount = false;
+    this.mainTxCats = ['tx', 'hash', 'block'];
+    this.txTableInit = <Text type="p">Must select a wallet and account to display transaction history</Text>;
+    this.txTableLoading = <Text type="p">Loading transaction history...</Text>;
+    // only grab the TXN_DISPLAY_COUNT most recent transactions
+    // TXN_DISPLAY_COUNT will never be too large of a number
+    this.negatedTx = ~TXN_DISPLAY_COUNT + 1;
```

Comment:
Good call on precomputing here

Reply:
Yeah, to highlight the point you were making, I wanted to check this value so console logged it out and it printed like 20 times 😁